### PR TITLE
BinarySearch.js Added

### PR DESCRIPTION
**Pull request by shubham-sutar**

### CODE REVIEW

File: BinarySearch.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+	return false;
+}
+
+// Driver code
+let arr = [1, 3, 5, 7, 8, 9];
+let x = 5;
+
+if (iterativeFunction(arr, x, 0, arr.length-1))
+	document.write("Element found!<br>");
```
Comment (harshshinde07) [inline BinarySearch.js]: Why are you using `documen.write` here?

**Comment by shubham-sutar** [comment]
Its JavaScript code bro.

File: BinarySearch.js (hunk 1 above)
Comment (shubham-sutar) [reply BinarySearch.js]: because this code has written in JavaScript.

Comment (harshshinde07) [reply BinarySearch.js]: Can you explain what it does? And why can't use use `console.log` instead?

**Comment by shubham-sutar** [comment]
console.log used to print the output on console and document.write used to
print output on desired screen template.
Both are used to print output but the purpose is different. plz accept PR
asap.

**Comment by harshshinde07** [comment]
Please convert it to `console.log` as we are not using any front end elements here so that should suffice.

**Comment by shubham-sutar** [comment]
> Please convert it to `console.log` as we are not using any front end elements here so that should suffice.

Let's check and merge PR asap. Thank You.. !!

**Review by harshshinde07 — APPROVED** [review]

**Pull request merged**